packages/dotnet-sdk/src/TeamsFx/Conversation/Channel.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace Microsoft.TeamsFx.Conversation
+{
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Schema;
+    using Microsoft.Bot.Schema.Teams;
+
+    /// <summary>
+    /// An <see cref="INotificationTarget"/> that represents a team channel.
+    /// </summary>
+    /// <remarks>
+    /// It's recommended to get channels from <see cref="TeamsBotInstallation.GetChannelsAsync"/>.
+    /// </remarks>
+    public class Channel : INotificationTarget
+    {
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        /// <param name="parent">The parent <see cref="TeamsBotInstallation"/> where this channel is created from.</param>
+        /// <param name="info">Detailed channel information.</param>
+        /// <exception cref="ArgumentNullException">Throws if provided parameter is null.</exception>
+        /// <remarks>
+        /// It's recommended to get channels from <see cref="TeamsBotInstallation.GetChannelsAsync"/>.
+        /// </remarks>
+        public Channel(TeamsBotInstallation parent, ChannelInfo info)
+        {
+            Parent = parent ?? throw new ArgumentNullException(nameof(parent));
+            Info = info ?? throw new ArgumentNullException(nameof(info));
+        }
+
+        /// <summary>
+        /// The parent <see cref="TeamsBotInstallation"/> where this channel is created from.
+        /// </summary>
+        public TeamsBotInstallation Parent { get; private set; }
+
+        /// <summary>
+        /// Detailed channel information.
+        /// </summary>
+        public ChannelInfo Info { get; private set; }
+
+        /// <summary>
+        /// The type of target. For channel it's always <see cref="NotificationTargetType.Channel"/>.
+        /// </summary>
+        public NotificationTargetType Type { get => NotificationTargetType.Channel; }
+
+        /// <inheritdoc/>
+        public Task SendMessage(string message, CancellationToken cancellationToken = default)
+        {
+            return Parent.Adapter.ContinueConversationAsync
+            (
+                Parent.BotAppId,
+                Parent.ConversationReference,
+                async (context1, ct1) => {
+                    var conversation = NewConversation(context1);
+                    await Parent.Adapter.ContinueConversationAsync
+                    (
+                        Parent.BotAppId,
+                        conversation,
+                        (context2, ct2) => context2.SendActivityAsync(message, cancellationToken: ct2),
+                        ct1
+                    ).ConfigureAwait(false);
+                },
+                cancellationToken
+            );
+        }
+
+        /// <inheritdoc/>
+        public Task SendAdaptiveCard(object card, CancellationToken cancellationToken = default)
```

Comment:
What if the schema of the card object is not a valid adaptive card, then how user troubleshooting with such error. Sometime Teams may have no response if the adaptive card content is incorrect

Reply:
I tried an invalid card object and Teams show an empty border:
<img width="369" alt="image" src="https://user-images.githubusercontent.com/7642967/167792209-1ba8cb9c-7b30-49e0-a16f-5b4f81711c43.png">

I believe the rendering logic is at Teams client side to hard to tell schema error here.
BTW, to provide any card validation utilities, I prefer using another sdk in user's code, not conversation sdk.

Reply:
Agree to provide card utility in a separate SDK, like "@microsoft/adaptiveCards-tools" for TS/JS.